src/site/content/en/accessible/labels-and-text-alternatives/self-assessment.assess.yml

```diff
@@ -3,10 +3,7 @@ setLeader: |
 height: unset
 tabLabel: sample
 questions:
-  - stimulus: |
-      ```html
-      <img src="temp.svg" alt="temp">
-      ```
+  - stimulus: "![temp](image/user/temp.svg)"
```

Comment:
I think this was changed by mistake

Reply:
This was done on purpose, I want to push authors to use markdown for the stimulus image as we have a mechanism in the assessment component to extract the markdown syntax to use the cdn.